<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MuseumGUI</class>
 <widget class="QDialog" name="MuseumGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>博物馆小灵</string>
  </property>
  <property name="windowIcon">
   <iconset resource="robotq.qrc">
    <normaloff>:/RobotQ/Resources/HIT_Robot_ico.ico</normaloff>:/RobotQ/Resources/HIT_Robot_ico.ico</iconset>
  </property>
  <widget class="QLabel" name="BG">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="robotq.qrc">:/MuseumGUI/Resources/博物馆背景.bmp</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRobotQ">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/语音交互.bmp</normaloff>:/MuseumGUI/Resources/语音交互.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>60</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoGuide">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>150</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/自动导航.bmp</normaloff>:/MuseumGUI/Resources/自动导航.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>60</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPath1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>400</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/路线1.bmp</normaloff>:/MuseumGUI/Resources/路线1.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>36</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPath2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/路线2.bmp</normaloff>:/MuseumGUI/Resources/路线2.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>36</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPath3">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>400</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/路线3.bmp</normaloff>:/MuseumGUI/Resources/路线3.bmp</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>36</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/Home.png</normaloff>:/MuseumGUI/Resources/Home.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>72</width>
     <height>72</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPath4">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/路线4.png</normaloff>:/MuseumGUI/Resources/路线4.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>36</height>
    </size>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/exit.png</normaloff>:/MuseumGUI/Resources/exit.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLock">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="robotq.qrc">
     <normaloff>:/MuseumGUI/Resources/unlocked.png</normaloff>:/MuseumGUI/Resources/unlocked.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="block1">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="block2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>390</y>
     <width>431</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="robotq.qrc"/>
 </resources>
 <connections/>
</ui>
